media-libs/libjxl/libjxl-9999.ebuild

```diff
@@ -0,0 +1,65 @@
+# Copyright 2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+inherit xdg cmake git-r3
+
+DESCRIPTION="JPEG XL image format reference implementation"
+HOMEPAGE="https://github.com/libjxl/libjxl"
+
+EGIT_REPO_URI="https://github.com/libjxl/libjxl.git"
+
+LICENSE="BSD"
+SLOT="0"
+IUSE="examples"
+
+DEPEND="app-arch/brotli
+	sys-libs/zlib
+	media-libs/libpng
+	virtual/jpeg
+	virtual/opengl
+	media-libs/freeglut
+	media-libs/giflib
+	media-libs/openexr:=
+	dev-util/google-perftools
+	x11-misc/shared-mime-info
+	dev-cpp/gtest
+	dev-cpp/highway
+	media-libs/lcms
+"
+
+BDEPEND=""
+
+RDEPEND="${DEPEND}"
+
+src_configure() {
+	local mycmakeargs=(
+		-DCMAKE_SKIP_RPATH=ON
+		-DBUILD_TESTING=OFF
+		-DBUILD_SHARED_LIBS=ON
+		-DJPEGXL_ENABLE_BENCHMARK=OFF
+		-DJPEGXL_ENABLE_COVERAGE=OFF
+		-DJPEGXL_ENABLE_FUZZERS=OFF
+		-DJPEGXL_ENABLE_SJPEG=OFF
+		-DJPEGXL_WARNINGS_AS_ERRORS=OFF
+
+		-DJPEGXL_ENABLE_SKCMS=ON
+		-DJPEGXL_ENABLE_EXAMPLES=$(usex examples)
+		-DJPEGXL_ENABLE_VIEWERS=OFF
+		-DJPEGXL_ENABLE_PLUGINS=ON
+		-DJPEGXL_FORCE_SYSTEM_BROTLI=ON
+		-DJPEGXL_FORCE_SYSTEM_HWY=ON
+		-DJPEGXL_FORCE_SYSTEM_GTEST=ON
```

Comment:
Is this needed with `-DBUILD_TESTING=OFF`?

Reply:
I would probably keep it even if we can't build tests for now for future work.

Reply:
Building those tests sometimes fail for weird reasons, takes time, that's why I disabled them.

Reply:
@thesamesam Do you have some example how it should look like?

Those tests are minefield. I'll expect troubles even if they tests are behind use flag.

Reply:
BTW, the problem with the tests was mentioned in https://bugs.gentoo.org/822444

Maybe you can look at it as I was unable to reproduce the problem and I do not know why it happened.